environment.sh

```diff
@@ -90,24 +91,35 @@ else
       # Empty string is no return on grep
       conda install jq -y
     fi
+    echo "Checking framework packages are up to date..."
     # Check if there has been an update to vivarium packages since last modification to requirements file
     # or more reccent than environment creation
-    grep @ $install_file 
-    # TODO: can we make this grep output a variable?
-    while read -r line ; do
-      # Parse each line of grep output
-      repo_info=(${line//@/ })
-      repo=${repo_info[0]}
-      repo_branch=${repo_info[2]}
-      last_commit_time=$(curl -H "Accept: application/vnd.github.v3+json" https://api.github.com/repos/ihmeuw/$repo/commits?sha=$repo_branch | jq '.["0"].commit.committer.date')
+    # Note: The lines we will return via grep will look like 'vivarium>=#.#.#' or will be of the format 
+    # 'vivarium @ git+https://github.com/ihmeuw/vivarium@SOME_BRANCH'
+    # echo $(grep -E 'vivarium|gbd|risk_distribution|layered_config' $install_file)
+    framework_packages=$(grep -E 'vivarium|gbd|risk_distribution|layered_config' $install_file)
+    echo "Grep output: $framework_packages"
+    num_packages=$(grep -E 'vivarium|gbd|risk_distribution|layered_config' -c $install_file)
+    echo "Number of framework packages: $num_packages"
+    for ((i = 1; i <= $num_packages; i++)); do
+      line=$(echo "$framework_packages" | sed -n "${i}p")
+      echo "Line is: $line"
+      # Check if the line contains '@'
+      if [[ "$line" == *"@"* ]]; then
+          repo_info=(${line//@/ })
```

Comment:
For some reason, I couldnt cut the line on the @ symbol the way I do with the > operator in the else branch. I'm assuming there is some sort of space/newline messing it up but this works.